pandapower/grid_equivalents/rei_generation.py

```diff
@@ -260,15 +261,15 @@ def _create_net_zpbn(net, boundary_buses, all_internal_buses, all_external_buses
             elm_idx = pp.create_sgen(net_zpbn, i, float(P), float(Q), name=key+"_rei_"+busstr,
                                      sn_mva=Sn, index=max_sgen_idx+len(net_zpbn.sgen)+1)
         elif elm == "gen":
-            vm_pu = v[key+"_vm_total"][v.ext_bus == int(busstr)].values.real
+            vm_pu = v[key+"_vm_total"][v.ext_bus == int(re.findall('\d+', busstr)[0])].values.real
```

Comment:
This is not a conform code

Reply:
> 

why? "busstr" is a string consisting of "buses" and "/", e.g., "25/45/89/33/1" or "48". The voltages for all these buses are the same. We only need one of them. The modification is to get the first number of the string.